LabExT/View/Controls/Plotting/PlotView.py

```diff
@@ -0,0 +1,119 @@
+        """Holds the subwindows."""
+
+        self._plotting_frame = PlottingFrame(master=self._paned_frame, figure=figure)
+
+        self._settings_frame = PlottingSettingsFrame(
+            master=self._paned_frame, axes=figure.axes[0], on_data_change=self._plotting_frame.data_changed_callback
+        )
+        self._settings_frame.title = "Plot Settings"
+
+    def show(self, row: int = 0, column: int = 1, width: int = 2, height=2, pad: int = 10) -> None:
+        """Places this element in a grid in the parent widget according to the arguments.
+        
+        Args:
+            row: row coordinate in the parent
+            width: columnspan in master
+            height: rowspan in master
+            column: column coordinate in the parent
+            pad: padding in x and y direction
+        """
+        self._paned_frame.add(self._plotting_frame, padx=pad, pady=pad, minsize=400, width=700)
+        self._paned_frame.add(self._settings_frame, padx=pad, pady=pad, minsize=200)
+
+        self._paned_frame.grid(
+            row=row, column=column, columnspan=width, rowspan=height, padx=pad, pady=pad, sticky="nsew"
+        )
+
+    def hide(self) -> None:
+        """Removes this element from the parent grid."""
+        self._paned_frame.remove(self._plotting_frame)
+        self._paned_frame.remove(self._settings_frame)
+        self._paned_frame.grid_forget()
+
+
+class PlottingFrame(tk.Frame):
+    """This frame contains the widgets created by the matplotlib tk backend"""
+
+    def __init__(self, master: tk.Widget, figure: Figure, *args, **kwargs) -> None:
+        """Initializes a new `PlottingWidget`
+
+        Args:
+            master: The parent this widget should be placed in
+            figure: The figure used to plot the data (provided by model)
+        """
+        super().__init__(master, *args, **kwargs)
+
+        self._canvas = FigureCanvasTkAgg(figure=figure, master=self)
+        self._canvas.get_tk_widget().pack(side=tk.TOP, fill=tk.BOTH, expand=True)
+        toolbar = NavigationToolbar2Tk(self._canvas, self)
+        toolbar.pack()
+
+    def data_changed_callback(self) -> None:
+        """This method should be called if the axes object was changed, such that the necessary updates can be performed."""
+        self._canvas.draw()
+
+
+class PlottingSettingsFrame(CustomFrame):
+    """This frame contains the widgets used to control the displayed visualization of the data"""
+
+    def __init__(self, master: tk.Widget, axes: Axes, on_data_change: Callable[[], None], *args, **kwargs) -> None:
+        super().__init__(master, *args, **kwargs)
+
+        self._data_changed_callback = on_data_change
+
+        self._value = 0
+        self._axes = axes
+        self._change_button = tk.Button(self, text="Click me", command=self.on_click)
+        self._change_button.pack(side=tk.TOP, fill=tk.BOTH)
+
+    def on_click(self):
+        self._axes.clear()
+        import numpy as np
```

Comment:
I would move the import to the top of the file to make the code clearer and to be conform with PEP8.

Reply:
Done in #200